### DISCUSSION TITLE
ASk : Sort & Limit

### DISCUSSION THREAD

**Discussion by zyahrial** [Kamva/mgm#60] [Q&A] [answered]
How to use sort and limit that using mgm for find documents

**Comment by mehran-prs** [comment, marked as answer]
Hi @zyahrial 

Same as the sort and limit in the MongoDB driver itself. e.g., 

```go
mgm.coll(&User{}).SimpleFindWithCtx(ctx, &models, bson.M{}, &options.FindOptions{
		Limit: &limit,
		Skip:  &offset,
	})

```

Use `FindOptions` in the `go.mongodb.org/mongo-driver/mongo/options` package.
